PROCEDURES.md

```diff
@@ -0,0 +1,199 @@
+
+Verify the release automation succeeded for all relevant repos via the Prow dashboard: [Prow Status for the release automation - all repos](https://prow.knative.dev/?job=release*).
+
+### Manually triggering a release
+You can manually trigger a release for a repository by re-running its release job.
+
+1. Navigate to https://prow.knative.dev/
+
+   ![Prow homepage](images/prow-home.png)
+
+2. Search for the `release*` job for the repository.
+
+   ![Search Prow for the repo and select the release*](images/prow-search.png)
+
+3. Rerun the release job.
+
+   ![Rerun Prow Release](images/prow-rerun.png)
+
+## Release notes
+### Exceptions
+Repos that don't require release artifacts (such as release tags and GitHub release for example) naturally don't need a release note and this step can be skipped for those. All of the [supporting repos](TIMELINE.md#supporting-repos) fit these criteria.
+
+### New release notes document
+For a new Knative release version, start a fresh HackMD release notes document by emptying out the [last release notes document](https://hackmd.io/cJwvzJ4eRVeqqiXzOPtxsA). Post and pin it to the **release-`xdotx`** Slack channel.
+
+### Release notes update
+After a repo release [has been published](#release-job), generate the repo's notes by using the `Release Notes` GitHub Action workflow. See an example in [Eventing](https://github.com/knative/eventing/actions?query=workflow%3A%22Release+Notes%22). The default starting and ending SHAs will work if running out of the `main` branch, or you can determine the correct starting and ending SHAs for the script to run.
+
+Use the generated notes to:
+
+1. add them to the [HackMD document](https://hackmd.io/cJwvzJ4eRVeqqiXzOPtxsA)
+2. ask the relevant WG leads if they have any add or edit for the updated repo notes in the [HackMD document](https://hackmd.io/cJwvzJ4eRVeqqiXzOPtxsA)
+3. use the GitHub UI to update the repo release tag with the final notes
+
+## Homebrew updates
+### homebrew-client
+After the client release, the [Homebrew tap](https://github.com/knative/homebrew-client) needs to be updated with the new release:
+
+- Copy `kn.rb` to the `kn@${PREV_RELEASE}.rb` with `$PREV_RELEASE` to be replace with the latest release (e.g. `0.19`).
+- In `kn@${PREV_RELEASE}.rb` replace `class Kn` with `class KnAT${PREV_RELEASE_DIGITS}`, e.g `class KnAT019` for an previous release `0.19`.
+- In `kn.rb`:
+  - Replace the old version number in `v` with the released version (e.g. `v = "v0.20.0"`)
+  - Replace the `sha256` checksums with the values from the [client release page](https://github.com/knative/client/releases). The checksums have been released too (e.g. [checksums.txt](https://github.com/knative/client/releases/download/v0.22.0/checksums.txt))
+
+✅ Open a PR and merge the changes. Prow is not enabled for the homebrew repo, so the merge needs to be performed manually.
+
+### homebrew-kn-plugins
```

Comment:
I just added a script to automate the version bump for the homebrew plugin formulas: https://github.com/knative-sandbox/homebrew-kn-plugins/blob/main/hack/update-versions.sh  Going to add one for the client formula also... hopefully makes this process a little easier